qemu/tests/cfg/verify_panic_status_with_pvpanic.cfg

```diff
@@ -0,0 +1,12 @@
+- verify_panic_status_with_pvpanic:
+    type = verify_panic_status_with_pvpanic
+    virt_test_type = qemu
+    monitors = 'qmp1'
+    monitor_type_qmp1 = qmp
+    stop_kdump_command = 'systemctl stop kdump'
+    trigger_crash = 'echo c > /proc/sysrq-trigger'
+    qmp_command = "query-status"
```

Comment:
Still need it?

Reply:
JOB ID     : 15f65511833efcbed2a99d8f20f4560c2cd444c6
JOB LOG    : /root/avocado/job-results/job-2020-11-23T00.17-15f6551/job.log
 (1/1) Host_RHEL.m8.u3.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.verify_panic_status_with_pvpanic.q35: PASS (70.07 s)